site/no-nb/index.html

```diff
@@ -0,0 +1,669 @@
+                            <li><a href="../index.html">English</a></li>
+                            <li><a href="de/index.html">Deutsch</a></li>
+                            <li><a href="el/index.html">Ελληνικά</a></li>
+                            <li><a href="es/index.html">Español</a></li>
+                            <li><a href="cn/index.html">中文</a></li>
+                            <li><a href="fr/index.html">Français</a></li>
+                            <li><a href="ja/index.html">日本語</a></li>
+                            <li><a href="">Norsk (bokmål)</a></li>
+                            <li><a href="pt-br/index.html">Português-Brasil</a></li>
+                            <li><a href="ru/index.html">Русский</a></li>
+                            <li><a href="hi/index.html">हिंदी</a></li>
+                        </ul>
+                    </li>
+                </ul>
+            </div>
+            <!-- /.navbar-collapse -->
+        </div>
+        <!-- /.container -->
+    </nav>
+
+
+    <!-- Header -->
+    <a class="anchor" name="about"></a>
+    <div class="intro-header">
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-push-2 col-lg-8">
+                    <div class="intro-message">
+                        <h1>Eclipse <span style="font-weight: 400">Collections</span></h1>
+                        <h3>- Funksjoner du ønsker med samlingene du trenger -</h3>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+        <!-- /.container -->
+
+    </div>
+    <!-- /.intro-header -->
+
+    <!-- Page Content -->
+
+    <a class="anchor" name="start"></a>
+    <div class="content-section-a">
+
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">Kom i gang med <br>Eclipse Collections</h2>
+                    <p class="lead">Eclipse Collections er det beste rammeverket for Java-samlinger 
+                        noensinne – det passer perfekt når du skal utvikle din Java-kode.
+                        <br><br>Integrer den i kodebasen din i dag med dine favorittbyggverktøy!</p>
+                    <object id="badge" data="https://snyk-widget.herokuapp.com/badge/mvn/org.eclipse.collections/eclipse-collections/badge.svg" type="image/svg+xml"></object>
+                </div>
+                <div class="col-lg-7 col-sm-6">
+                    <ul class="nav nav-tabs">
+                        <li class="active"><a href="#maven" data-toggle="tab">Maven</a></li>
+                        <li><a href="#gradle" data-toggle="tab">Gradle</a></li>
+                        <li><a href="#ivy" data-toggle="tab">Ivy</a></li>
+                        <li><a href="#download" data-toggle="tab">Nedlasting</a></li>
```

Comment:
I would rather say "Last ned", to use a more active language.

Reply:
Agreed! Active voice is better! Tnx